geoAssembler/tests/test_centre_optimiser.py

```diff
@@ -31,7 +31,7 @@ def test_integrator():
     assert 827 < misaligned_2dint_r.shape[0] < 837
     assert 952 < misaligned_2dint_a.shape[0] < 962
 
-    misaligned_1dint = np.nanmean(misaligned_2dint, axis=0)[100:-100]
```

Comment:
@RobertRosca please check that I've got this right - the idea is that slicing the data along axis 1 before averaging along axis 0 is logically the same as averaging and then slicing, but avoids trying to take an average of an all-nan column, which I think gives the warning.

Reply:
Hmm had to think about it a bit but yes that sounds correct I think.